src/Orm/BaseModel/index.ts

```diff
@@ -1888,7 +1888,7 @@ export class BaseModel implements LucidRow {
 	/**
 	 * Reload/Refresh the model instance
 	 */
-	public async refresh() {
+	public async refresh(options?: ModelAdapterOptions) {
```

Comment:
We can make use of the existing `$trx` property vs accepting it via options

Reply:
I've updated